Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -79,7 +79,7 @@
                 }
                 else
                 {
-                    <_DataGridRow @key="@item" TItem="TItem" Item="@item" Columns="@Columns" Edit="@OnEditCommand" Delete="@OnDeleteCommand" Save="@OnSaveCommand" Cancel="@OnCancelCommand" Selected="@OnSelectedCommand" />
+                    <_DataGridRow @key="@item" TItem="TItem" Item="@item" Columns="@Columns" Style="@(RowSelectable?.Invoke(item) ?? false ? "cursor: pointer" : "")" Edit="@OnEditCommand" Delete="@OnDeleteCommand" Save="@OnSaveCommand" Cancel="@OnCancelCommand" Selected="@OnSelectedCommand" />
```

Comment:
So, I tried the PR and this line is not working as expected. First, the `Style `is not doing anything because it's not setting anything inside of `_DataGridRow`. The Style needs to be passed to `TableRow` or `TableRowCell` to make it work. But I would rather remove Style and implement `Cursor` attribute on `TableRow` or `TableRowCell`. You can see the similar code on `CheckEdit` component.

Reply:
Also. `RowSelectable` should be true by default instead of false.

Reply:
> So, I tried the PR and this line is not working as expected. First, the `Style `is not doing anything because it's not setting anything inside of `_DataGridRow`. The Style needs to be passed to `TableRow` or `TableRowCell` to make it work.

Indeed, `Style` is the wrong property here, you're right - thanks.
`TableRow` or `TableRowCell` - good question. Cell would be better, in case anybody wants to override the cursor for a specific cell?

> But I would rather remove Style and implement `Cursor` attribute on `TableRow` or `TableRowCell`. You can see the similar code on `CheckEdit` component.

Sounds good to me. So we'd pass `_DataGridRow.Cursor` the cursor it should show here. Is there an enum for the cursors? Or want me to pass the strings? `_DataGridRow.razor` would pass it along to the `TableRow` or `TableRowCell` then.

> Also. `RowSelectable` should be true by default instead of false.

Do you mean the `?? false ?` part of my changes?
If so, my thoughts were that I don't want to introduce a "breaking change" for the cursor in the `DataGrid` component.

### Current behavior:
_RowSelectable is `null`_: normal cursor
_RowSelectable returns `false`_: normal cursor
_RowSelectable returns `true`_: normal cursor

### New behavior (non-breaking in this PR):
_RowSelectable is `null`_: normal cursor
_RowSelectable returns `false`_: normal cursor
_RowSelectable returns `true`_: pointer cursor

### Regarding your comment (if I got that right, "breaking change"):
**_RowSelectable is `null`_: pointer cursor** ("breaking change")
_RowSelectable returns `false`_: normal cursor
_RowSelectable returns `true`_: pointer cursor

Reply:
You can see the Cursor enum in here: https://github.com/stsrki/Blazorise/blob/4b0917bad042eb863640c247bdea4134fafdc816/Source/Blazorise/Enums.cs#L1099

I know it's a little more work there but it's the right way to do :)

Regarding the RowSelectable. It's already working as a selectable even if the RowSelectable is null. So I think it you can go with:

RowSelectable is null: pointer cursor ("not really a breaking change")
RowSelectable returns false: normal cursor
RowSelectable returns true: pointer cursor

Reply:
Just to let you know. When you add cursor styles for table row/cell you add them to `[Blazorise.ProviderName]/Styles/providerName.scss`, [example here](https://github.com/stsrki/Blazorise/blob/4b0917bad042eb863640c247bdea4134fafdc816/Source/Blazorise.Bootstrap/Styles/blazorise.bootstrap.scss#L21)

Then for it to be compiled automatically you need to have VS extension [WebCompiler](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebCompiler). That way it will compile it to the final css.